strawberry/arguments.py

```diff
@@ -137,22 +207,23 @@ def convert_arguments(
     kwargs = {}
 
     for argument in arguments:
-        if argument.name in value:
-            origin_name = cast(str, argument.origin_name)
-            current_value = value[argument.name]
+        if argument.graphql_name in value:
+            assert argument.python_name
+
+            current_value = value[argument.graphql_name]
 
-            kwargs[origin_name] = convert_argument(current_value, argument)
+            kwargs[argument.python_name] = convert_argument(current_value, argument)
 
     return kwargs
 
 
-def argument(description: Optional[str] = None) -> StrawberryArgument:
-    return StrawberryArgument(description=description)
+def argument(description: Optional[str] = None) -> StrawberryArgumentAnnotation:
+    return StrawberryArgumentAnnotation(description=description)
 
 
+# TODO: check exports
```

Comment:
We might want to create an issue to just look at imports/exports project wide and make sure they are consistent

Reply:
Done :) https://github.com/strawberry-graphql/strawberry/issues/806